SU-SWS/ace-casbsgryphon
# Cas 181 improve styling temp

# NOT READY FOR REVIEW

# Summary
- This is just to fix some styling for launch. It is terrible. My subtheme (which is my preferred approach) is not compiling. Don't judge me too harshly!

# Review By (Date)
- Today
- After the last PR related to CAS-180 is merged

# Urgency
- Important

# Steps to Test

1. Go to a people page like people/current-fellows and notice that images are left-aligned and the jumplist is styled
2. Go to the recent work page and notice the same thing about the image  /about/recent-works-fellows

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- CAS-180

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
